hasher-matcher-actioner/mypy.ini

```diff
@@ -1,26 +1,6 @@
-[mypy]
```

Comment:
This was just to suppress a warning from `python -m mypy hmalib` if we don't have it

Reply:
Got it. Will add back